Apps/Sandcastle/gallery/Scene Rendering Performance.html

```diff
@@ -0,0 +1,283 @@
+    }));
+
+    tileset.readyPromise.then(function() {
+        var boundingSphere = tileset.boundingSphere;
+        viewer.camera.viewBoundingSphere(boundingSphere, new Cesium.HeadingPitchRange(0.0, -0.5, boundingSphere.radius));
+        viewer.camera.lookAtTransform(Cesium.Matrix4.IDENTITY);
+    }).otherwise(function(error) {
+        throw(error);
+    });
+}
+
+// Load CZML DataSource with a model and set the trackedEntity.
+// No need to call scene.requestRender()
+// Enable and adjust maximum simulation time change to see
+// animations at desired speed.
+function loadCzmlScenario() {
+    resetScene();
+
+    var czml = [{
+        "id" : "document",
+        "name" : "CZML Model",
+        "version" : "1.0"
+    }, {
+        "id" : "aircraft model",
+        "name" : "Cesium Air",
+        "position" : {
+            "cartographicDegrees" : [-77, 37, 10000]
+        },
+        "model": {
+            "gltf" : "../../SampleData/models/CesiumAir/Cesium_Air.glb",
+            "scale" : 2.0,
+            "minimumPixelSize": 128
+        }
+    }];
+
+    var dataSourcePromise = viewer.dataSources.add(Cesium.CzmlDataSource.load(czml));
+
+    dataSourcePromise.then(function(dataSource){
+        viewer.trackedEntity = dataSource.entities.getById('aircraft model');
+    });
+}
+
+// Pick an entity, only rendering when needed.
+function pickingScenario() {
+    resetScene();
+
+    var entity = viewer.entities.add({
+        position : Cesium.Cartesian3.fromDegrees(-75.59777, 40.03883),
+        box : {
+            dimensions : new Cesium.Cartesian3(1000000.0, 1000000.0, 30000.0),
+            material : Cesium.Color.CORNFLOWERBLUE
+        }
+    });
+
+    // If the mouse is over the box, change its scale and color,
+    // then request a new render frame.
+    var lastPicked;
+    handler = new Cesium.ScreenSpaceEventHandler(scene.canvas);
+    handler.setInputAction(function(movement) {
+        var pickedObject = scene.pick(movement.endPosition);
+        if (Cesium.defined(pickedObject) && (pickedObject.id === entity)) {
+            if (Cesium.defined(lastPicked)) {
+                return;
+            }
+
+            entity.box.material = Cesium.Color.YELLOW;
+            scene.requestRender();
+            lastPicked = pickedObject;
+        } else if (Cesium.defined(lastPicked)) {
+            entity.box.material = Cesium.Color.CORNFLOWERBLUE;
+            scene.requestRender();
+            lastPicked = undefined;
+        }
+    }, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
+}
+
+// Changes to the scene with the API will require
+// calling requestRender() on change.
+function setScenePropertiesScenario() {
+    resetScene();
+
+    scene.skyBox.show = false;
+    scene.backgroundColor = Cesium.Color.CORNFLOWERBLUE;
+    scene.requestRender();
+}
+
+// BEGIN SANDCASTLE EXAMPLE UI SETUP
+
+var viewModel = {
+    requestRenderMode : true,
+    timeChangeEnabled : false,
+    maximumRenderTimeChange : 30.0,
```

Comment:
This default is too high, I would think the common case would be  in the 0-1 range if time is moving forward.  Maybe make the slider between 1-5 or 1-10

Reply:
I've adjusted the simulation time to be pretty high so the user can obviously see changes in the scene when a new render is requested. I can reset the clock and set this to a more common case if that would be more helpful.

Reply:
Um, I think the way the demo is set up right now just makes these options confusing.  You have to both start the animation and be on an example that has an animation to see what the difference is.  Can you add a new example and only show these panel options when that example is selected?  Use something like simple.czml that is always changing in an obvious way